nibabel/spatialimages.py

```diff
@@ -321,9 +322,99 @@ class ImageDataError(Exception):
     pass
 
 
+class SpatialFirstSlicer(object):
+    ''' Slicing interface that returns a new image with an updated affine
+
+    Checks that an image's first three axes are spatial
+    '''
+    def __init__(self, img):
+        from .imageclasses import spatial_axes_first
```

Comment:
Can this go at the top of the file now?  Or would that make a circular import?  If so, maybe worth a comment.